docker/config/test.env

```diff
@@ -3,6 +3,26 @@
 # put filesystem crashstorage stuff in /tmp/test_crashes
 resource.fs.fs_root=/tmp/test_crashes
 
+# Statsd
+STATSD_CLIENT=django_statsd.clients.null
+
+# Sentry
+SENTRY_DSN=
+RAVEN_DSN=
+
+# Boto
+AWS_ACCESS_KEY=something
+AWS_SECRET_ACCESS_KEY=something
+# NOTE(willkg): We want boto to use connect_to_region, so we make this an
+# empty string
+AWS_HOST=
+# Use test buckets
+resource.boto.bucket_name=crashstats-test
+TELEMETRY_BUCKET_NAME=telemetry-test
+
+# Elasticsearch configuration
+ELASTICSEARCH_URLS=http://example.com:9123
```

Comment:
I moved some configuration from `webapp-django/crashstats/settings/test.py` into this `test.env` file. We have multiple places for test configuration and I want to keep pushing to reduce it further.